gossipd/gossip.c

```diff
@@ -1927,7 +1934,11 @@ static struct io_plan *handle_disable_channel(struct io_conn *conn,
 			      strerror(errno));
 	}
 
-	handle_channel_update(daemon->rstate, msg);
+	err = handle_channel_update(daemon->rstate, msg);
```

Comment:
Same here, if we can't be trusted to disable tell ourselves about a channel we disable, then that's on us.